.github/workflows/rust.yml

```diff
@@ -39,7 +41,7 @@ jobs:
     - name: Run tests
       run: cargo test --verbose
   build_result:
-    name: homu build finished
+    name: Result
     runs-on: ubuntu-latest
     needs:
```

Comment:
Do we need to include the other jobs(msrv & semver) here in 'needs' clause?

Reply:
Yeah, it looks like we might need that.